Gulpfile.js

```diff
@@ -216,3 +216,9 @@ gulp.task('test-client', ['build'], function () {
 });
 
 gulp.task('test', ['lint', 'test-server']);
+
+gulp.task('error-message-lister', function () {
```

Comment:
Maybe `report-designer` or smthg? It'll be nice to have README in this dir as well, which describes why we need this utility. \cc @VasilyStrelyaev